strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/io/ArtifactOutputStreamTest.java

```diff
@@ -42,13 +42,22 @@
 {
 
     public static final String REPOSITORY_RELEASES = "aos-releases";
+    
+    private static boolean initialized;
```

Comment:
>org.carlspring.strongbox.testing.TestCaseWithRepository#removeRepositoriesdoes not remove the repositories physically, so they shouldn't be created once again in the org.carlspring.strongbox.services.MavenChecksumServiceTest#initialize method again (regardless initializedflag).

The thing is that test shouldn't depend on each other. We will go mad with such dependent test chains.